doc/manuals/user/federation.md

```diff
@@ -33,7 +33,8 @@ Next, let's send a subscribeContext to A (to make B subscribe to updates
 made in A). Note that the URL used in the reference has to be
 "/v1/notifyContext":
 
-    (curl localhost:1030/v1/subscribeContext -s -S --header 'Content-Type: application/xml' -d @- | xmllint --format -) <<EOF
+    (curl localhost:1030/v1/subscribeContext -s -S --header 'Content-Type: application/xml' \ 
```

Comment:
Not really fixed... at leats, github diff stills showing this in the "old" style.